statsmodels/regression/heckman.py

```diff
@@ -0,0 +1,666 @@
+            if isinstance(zname, basestring):
+                zname = [zname]
+
+
+        ## create summary object
+        # instantiate the object
+        smry = summary.Summary()
+
+        # add top info
+        if self.method=='twostep':
+            methodstr = 'Heckman Two-Step'
+        elif self.method=='mle':
+            methodstr = 'Heckman MLE'
+        else:
+            raise ValueError("Invalid method set")
+
+        top_left = [('Dep. Variable:', None),
+                    ('Model:', None),
+                    ('Method:', [methodstr]),
+                    ('Date:', None),
+                    ('Time:', None),
+                    ('No. Total Obs.:', ["%#i" % self.model.nobs_total]),
+                    ('No. Censored Obs.:', ["%#i" % self.model.nobs_censored]),
+                    ('No. Uncensored Obs.:', ["%#i" % self.model.nobs_uncensored]),
+                    ]
+
+        if hasattr(self, 'cov_type'):
+            top_left.append(('Covariance Type:', [self.cov_type]))
+
+        top_right = [
+                     ]
+
+        if title is None:
+            title = self.model.__class__.__name__ + ' ' + "Regression Results"
+
+        smry.add_table_2cols(self, gleft=top_left, gright=top_right,
+                          yname=yname, xname=xname, title=title)
+        
+        # add the Heckman-corrected regression table
+        smry.add_table_params(self, yname=yname, xname=xname, alpha=alpha,
+                             use_t=self.use_t)
+        
+        # add the selection equation estimates table
+        smry.add_table_params(self.select_res, yname=yname, xname=zname, alpha=alpha,
+                             use_t=self.select_res.use_t)
+
+        # add the estimate to the inverse Mills estimate
+        smry.add_table_params(
+            base.LikelihoodModelResults(None, np.atleast_1d(self.param_inverse_mills), 
+            normalized_cov_params=np.atleast_1d(self.stderr_inverse_mills**2), scale=1.), 
+            yname=None, xname=['IMR (Lambda)'], alpha=alpha, 
+            use_t=False)  #TODO: return t-score instead of z-score for IMR
+        
+        # add point estimates for rho and sigma
+        diagn_left = [('rho:', ["%#6.3f" % self.corr_eqnerrors]),
+                      ('sigma:', ["%#6.3f" % np.sqrt(self.var_reg_error)]),
+                      ]
+
+        diagn_right = [
+                       ]
+
+        smry.add_table_2cols(self, gleft=diagn_left, gright=diagn_right,
+                          yname=yname, xname=xname,
+                          title="")
+
+        # add text at end
+        smry.add_extra_txt(['First table are the estimates for the regression (response) equation.',
+            'Second table are the estimates for the selection equation.',
+            'Third table is the estimate for the coef of the inverse Mills ratio (Heckman\'s Lambda).'])
+
+        ## Print summary if option set to do so
+        if(disp):
+            print(smry)
```

Comment:
following my usual pattern in the notebooks, this prints two copies of the summary.

delete to be consistent across models.  
(using False as default may be possible, but then we should have a reason to add this across models.)